server.js

```diff
@@ -32,6 +36,12 @@ app.get("/users/:id/responders", middleware.checkToken, user.getResponders);
 app.get("/users/:id/responders/count",middleware.checkToken,user.getResponderCount);
 app.delete("/users/:id/responders/:responderid", middleware.checkToken, user.deleteResponder);
 
+// Help requests
+app.post("/help_requests", middleware.checkToken, help_request.addHelpRequest);
```

Comment:
Is it weird to have underscores in URLs?
```suggestion
app.post("/help-requests", middleware.checkToken, help_request.addHelpRequest);
```